src/pages/singleVideoPage/singelVideoPage.jsx

```diff
@@ -0,0 +1,82 @@
+import MenuIcon from "@mui/icons-material/Menu";
+import { Search } from "../../components/searchBox/searchbox";
```

Comment:
Use absolute imports here makes it clean and less confusing.

Reply:
ok, will optimize